apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -24,31 +29,131 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 		[handleSubmit, onSubmit],
 	);
 
+	type InputType = {
+		icon: string;
+		type: "password" | "text";
+		value: string;
+	};
+
+	const [passwordData, setPasswordData] = useState<InputType>({
+		icon: slashEyeIcon,
+		type: "password",
+		value: "",
+	});
+
+	const [confirmPasswordData, setConfirmPasswordData] = useState<InputType>({
+		icon: slashEyeIcon,
```

Comment:
You should use react-hook-form for all forms. For now no need to add icon to show/hide password